packages/@okta/vuepress-site/conductor.yml

```diff
@@ -2422,7 +2422,7 @@ redirects:
   - from: /docs/guides/add-an-external-idp/-/next-steps/index.html
     to: /docs/guides/add-an-external-idp/-/main/#next-steps
   - from: /docs/guides/add-an-external-idp/-/configure-idp-in-okta/#social-identity-provider-settings
-    to: /docs/concepts/identity-providers/#account-linking-and-just-in-time-provisioning
+    to: /docs/guides/social-login/
```

Comment:
Something weird is going on. When you enter the from, I'm landing here: /docs/guides/add-an-external-idp/openidconnect/main/#create-an-identity-provider-in-okta